bundle/edu.gemini.pot/src/test/scala/edu/gemini/spModel/target/env/GuideGroupSpec.scala

```diff
@@ -101,20 +99,20 @@ class GuideGroupSpec extends Specification with ScalaCheck with Arbitraries {
       }
 
     "remove the primary guide star if there is no primary in GuideProbeTargets" in
-      forAll { (g: GuideGroup) =>
+      forAll { g: GuideGroup =>
         AllProbes.forall { gp =>
           val noPrimaryGpt = GuideProbeTargets.create(gp, new SPTarget()).clearPrimarySelection()
-          val g2           = g.put(noPrimaryGpt)
-          g2.get(gp).asScalaOpt.forall { _.getPrimary.isEmpty }
+          val g2 = g.put(noPrimaryGpt)
+          g2.get(gp).asScalaOpt.forall(_.getPrimary.isEmpty)
         }
       }
 
     "remove the guider from automatic groups if there is no primary in GuideProbeTargets" in
-      forAll { (g: GuideGroup) =>
+      forAll { g: GuideGroup =>
         AllProbes.forall { gp =>
           val noPrimaryGpt = GuideProbeTargets.create(gp, new SPTarget()).clearPrimarySelection()
-          val g2           = g.put(noPrimaryGpt)
-          val gpt2         = g2.get(gp).asScalaOpt
+          val g2 = g.put(noPrimaryGpt)
+          val gpt2 = g2.get(gp).asScalaOpt
```

Comment:
Trivial again, but I think series of assignments are generally easier to read when the `=` signs line up.